#tr1:
  joint_limits:

    JointRightShoulderPan:
      #has_position_limits: true
      has_position_limits: false
      min_position: -1.0
      max_position: 1.5708
      #has_velocity_limits: true
      has_velocity_limits: false
      max_velocity: 0.2932
      has_acceleration_limits: false
      max_acceleration: 0.0
      #has_jerk_limits: true
      has_jerk_limits: false
      max_jerk: 100.0
      #has_effort_limits: true
      has_effort_limits: false
      max_effort: 1.0
      min_effort: -1.0
